dist/accordion/css/accordion.min.css

```
@@ -1 +1,62 @@
-.aria-accordion__panel[hidden]{display:none}
\ No newline at end of file
+/* For legacy browsers support */
+.aria-accordion__panel[hidden] {
+  display: none;
+}
+
+/* For accordion panels with a transition effect */
+.aria-accordion__panel--transition {
+  display: block;
+  max-height: 100vh;
```

```suggestion
  max-height: 80vh;
```

I tested with `80vh` as value and it ensures the visibility for both **Accordion Header** and its associated **Accordion Panel** inside the available viewport in case of very very long content.